jumeg_causality/dirs_manage.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Hi Praveen,

Yes we can integrate and simplify it.

Best wishes,
Qunxi Dong

2015-08-25 16:25 GMT+02:00 Praveen Sripad <notifications@github.com>:

> In jumeg_causality/dirs_manage.py
> <https://github.com/jdammers/jumeg/pull/94#discussion_r37870863>:
>
> > @@ -0,0 +1,24 @@
> > +import os
>
> @dongqunxi <https://github.com/dongqunxi> Can we put all of these
> functions into jumeg_utils.py file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdammers/jumeg/pull/94/files#r37870863>.
>